src/frontend/js/data/getResourceList/index.ts

```diff
@@ -17,14 +17,11 @@ export async function fetchList(
   params: APIListRequestParams = API_LIST_DEFAULT_PARAMS,
 ): Promise<FetchListResponse> {
   try {
-    const response = await fetch(
-      `/api/v1.0/${kind}/?${stringify(params, { arrayFormat: 'comma' })}`,
-      {
-        headers: {
-          'Content-Type': 'application/json',
-        },
+    const response = await fetch(`/api/v1.0/${kind}/?${stringify(params)}`, {
+      headers: {
+        'Content-Type': 'application/json',
       },
-    );
+    });
```

Comment:
I'm curious, why didn't the linter catch it?

Reply:
Because, there was a comma before.
It acts like black linter with tuple in python
`('item1',)` is not formatted but `('item1')` is formatted into `'item1'`